mlir/lib/CAPI/RegisterRocMLIR/RegisterRocMLIR.cpp

```diff
@@ -10,11 +10,24 @@
 #include "mlir-c/RegisterRocMLIR.h"
 
 #include "mlir/CAPI/IR.h"
+#include "mlir/Pass/PassManager.h"
 #include "mlir/InitRocMLIRDialects.h"
 #include "mlir/InitRocMLIRPasses.h"
+#include "llvm/Support/CommandLine.h"
 
 void mlirRegisterRocMLIRDialects(MlirDialectRegistry registry) {
   mlir::registerRocMLIRDialects(*unwrap(registry));
 }
 
 void mlirRegisterRocMLIRPasses() { mlir::registerRocMLIRPasses(); }
+
+void mlirRegisterRocMLIROptions() {
```

Comment:
The open question is, to call this function from MIGraphX, in mlir.cpp where we also call the other two register functions in this file, or call it from one of the other two so we don't have to patch the other project.

Reply:
I'd say patch MIGraphX.